dex/networks/eth/tokens.go

```diff
@@ -125,6 +125,61 @@ var Tokens = map[uint32]*Token{
 			},
 		},
 	},
+	usdcTokenID: {
+		EVMFactor: new(int64),
+		Token: &dex.Token{
+			ParentID: EthBipID,
+			Name:     "USDC",
+			UnitInfo: dex.UnitInfo{
+				AtomicUnit: "sixth",
```

Comment:
Not centi-centi-cent? 🤣

Maybe `microUSD`?

Reply:
Changed to microUSD